.github/workflows/ci.yml

```diff
@@ -78,6 +79,8 @@ jobs:
       # point out linter violations).
       - name: Lint
         uses: golangci/golangci-lint-action@v3
+        env:
+           RUNNING_IN_CI: true
```

Comment:
@fahedouch how does this env var modify lint behavior?

Reply:
fixed